server/api/reingest/reingest.controller.ts

```diff
@@ -0,0 +1,131 @@
+import {Response} from 'express';
+import {ApiRequest, EdipiParam} from '../index';
+import config from '../../config';
+import AWS from 'aws-sdk';
+
+class ReingestController {
+
+  static max_payload_size = 100;
+
+  async reingestSymptomRecords(req: ApiRequest<EdipiParam, TimestampData>, res: Response) {
+
+    let startDate = 0;
+    let endDate = Date.now();
+    if (req.body.startTime) {
+      startDate = ReingestController.convertDateParam(req.body.startTime);
+    }
+
+    if (req.body.endTime) {
+      endDate = ReingestController.convertDateParam(req.body.endTime);
+    }
+
+    const dynamoDB = new AWS.DynamoDB();
+    const queryInput = {
+      TableName: config.dynamo.symptomTable,
+      IndexName: config.dynamo.symptomIndex,
+      KeyConditions: {
+        "EDIPI": {
+          ComparisonOperator: "EQ",
+          AttributeValueList: [
+            {
+              S: req.params.edipi,
+            },
+          ],
+        },
+        "Timestamp": {
+          ComparisonOperator: "BETWEEN",
+          AttributeValueList: [
+            {
+              N: startDate.toString(),
+            },
+            {
+              N: endDate.toString(),
+            },
+          ],
+        },
+      },
+
+    };
+
+    try {
+      const data = await dynamoDB.query(queryInput).promise();
+
+      const {payloads, recordsIngested} = ReingestController.buildPayloadsFromData(data);
+
+      const lambda = new AWS.Lambda();
+      let lambdaInvocationCount = 0;
+      for (const lambdaPayload in payloads) {
+        const lambdaParams = {
+          FunctionName: config.dynamo.symptomLambda,
+          Payload: Buffer.from(JSON.stringify(lambdaPayload)),
+          InvocationType: 'Event'
+        }
+        lambda.invoke(lambdaParams);
+        lambdaInvocationCount++;
+      }
+
+      res.status(201).json({
+        'reingest-count': recordsIngested,
+        'lambda-invoke-count': lambdaInvocationCount
+      });
+    } catch(err) {
+      console.log('Reingest Failed: ', err.message);
+      res.status(500).send(`Error: ${err.message}`);
```

Comment:
We have been avoiding manually defining error responses.  You can throw a `InternalServerError` with the message and it will be handled appropriately.